app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala

```diff
@@ -39,23 +42,41 @@ object AdditionalFiscalReferences {
 }
 
 case class ItemType(
-                     combinedNomenclatureCode: String,
-                     taricAdditionalCode: Seq[String],
-                     nationalAdditionalCode: Seq[String],
-                     descriptionOfGoods: String,
-                     cusCode: Option[String],
-                     unDangerousGoodsCode: Option[String],
-                     statisticalValue: String
+  combinedNomenclatureCode: String,
+  taricAdditionalCode: Seq[String],
+  nationalAdditionalCode: Seq[String],
+  descriptionOfGoods: String,
+  cusCode: Option[String],
+  unDangerousGoodsCode: Option[String],
+  statisticalValue: String
 )
 object ItemType {
   implicit val format: OFormat[ItemType] = Json.format[ItemType]
 }
 
-case class PackageInformation(
-  typesOfPackages: Option[String],
-  numberOfPackages: Option[Int],
-  shippingMarks: Option[String]
-)
+sealed abstract class IdentificationTypeCodes(val value: String)
```

Comment:
Similar here, if we used Scala Enumerations this would be a lot less code

Reply:
as mentioned in another comment...